packages/cli-repl/test/e2e-analytics.spec.ts

```diff
@@ -16,31 +16,31 @@ describe('e2e Analytics', () => {
     await TestShell.killall();
   });
 
-  context('setting up the replica set', () => {
-    it('works', async() => {
-      const rsConfig = {
-        _id: replSetName,
-        members: [
-          { _id: 0, host: `${await rs0.hostport()}`, priority: 2 },
-          { _id: 1, host: `${await rs1.hostport()}`, priority: 1 },
-          { _id: 2, host: `${await rs2.hostport()}`, priority: 1 },
-          { _id: 3, host: `${await rs3.hostport()}`, priority: 0, votes: 0, tags: { nodeType: 'ANALYTICS' } }
-        ]
-      };
+  before(async() => {
+    const rsConfig = {
+      _id: replSetName,
+      members: [
+        { _id: 0, host: `${await rs0.hostport()}`, priority: 2 },
+        { _id: 1, host: `${await rs1.hostport()}`, priority: 1 },
+        { _id: 2, host: `${await rs2.hostport()}`, priority: 1 },
+        { _id: 3, host: `${await rs3.hostport()}`, priority: 0, votes: 0, tags: { nodeType: 'ANALYTICS' } }
+      ]
+    };
 
-      const shell = TestShell.start({
-        args: [await rs0.connectionString()]
-      });
-      await shell.waitForPrompt();
-      await shell.executeLine(`rs.initiate(${JSON.stringify(rsConfig)})`);
-      shell.assertContainsOutput('ok: 1');
-      await eventually(async() => {
-        const isMaster = await shell.executeLine('db.isMaster().ismaster');
-        expect(isMaster).to.contain('true');
-      });
+    const shell = TestShell.start({
+      args: [await rs0.connectionString()]
+    });
+    await shell.waitForPrompt();
+    await shell.executeLine(`rs.initiate(${JSON.stringify(rsConfig)})`);
+    shell.assertContainsOutput('ok: 1');
+    await eventually(async() => {
+      const isMaster = await shell.executeLine('db.isMaster().ismaster');
+      expect(isMaster).to.contain('true');
     });
+  });
```

Comment:
Unrelated, but without this one can’t run one of the other individual tests in this file